doc/developer-guides/hld/hv-console.rst

```diff
@@ -70,10 +70,17 @@ Specifically:
    the hypervisor shell. Inputs to the physical UART will be
    redirected to the vUART starting from the next timer event.
 
--  The vUART is deactivated after a :kbd:`Ctrl` + :kbd:`Space` hotkey is received
-   from the physical UART. Inputs to the physical UART will be
-   handled by the hypervisor shell starting from the next timer
-   event.
+-  The vUART enters escaping mode after a break key (:kbd:`Ctrl` +
+   :kbd:`Space` hotkey) is received from the physical UART. When in escaping
+   mode, next received character will be a command. After processing this
+   command, the vUART leaves the escaping mode. So far, following escaping
+   commands are supported:
+
+   - Break key. If user sends break key again in escaping mode, i.e. press
+     :kbd:`Ctrl` + :kbd:`Space` twice, one break key will be send to vUART.
+
+   - Character "e". This will deactive vUART. Inputs to the physical UART will
+     be handled by the hypervisor shell starting from the next timer event.
```

Comment:
We shall also mention what happens if a different character is given. Are such characters sent to the vUART as if the escaping never happens, or are those silently ignored?

Reply:
Sorry, missed. It will prompt a unsupported escaping command message and exit escaping mode.